xlml/utils/metric.py

```diff
@@ -355,11 +443,16 @@ def generate_process_id() -> str:
   return str(uuid.uuid4())
 
 
-def is_valid_entry() -> bool:
+def is_valid_entry(task_metric_config: metric_config.MetricConfig) -> bool:
```

Comment:
Oh I see, no worries! Do you mean to remove the lines mentioned and only allow runs in the prod env `ml-automation-solutions`? How about collecting metrics in individual development environments?

Reply:
If you remove those lines (https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/xlml/utils/metric.py#L368-L373), we enabled metric collections for manual runs in prod. So basically all metrics from runs in prod stage will be sent to datasets. I think this is also what you want right? still no metrics from non-prod runs

Reply:
I would also want to collect metrics in development composer environment. For example if I want to run a sweep of jobs in my own environment and get metrics for it

Reply:
I understand that. I think the right way for this is to have a separate dev dataset setup for dev composer env. We shouldn't pollute dev data into prod stage.

Reply:
That sgtm, I changed it so that all runs in the prod env (including manual runs) will write to bigquery and then any env can write to a separate dev dataset `benchmark_dataset_dev`. Does this look good?

Reply:
I removed the changes to `is_valid_entry()`. Will follow the changes from https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/112 as discussed offline.